common/mariadb/templates/alerts/_backup-v2.alerts.tpl

```diff
@@ -1,7 +1,7 @@
 - name: backup-v2.alerts
   rules:
   - alert: {{ include "alerts.service" . | title }}MariaDatabaseFullBackupMissing
-    expr: maria_backup_status{kind="full_backup", release={{ include "alerts.service" . | quote }} } != 1
+    expr: maria_backup_status{kind="full_backup", app_kubernetes_io_instance=~"{{ include "fullName" . }}" } != 1
```

Comment:
currently we upload these backups only to swift, without aws, so maybe we should add swift label filter here